client/lib/cart/store/cart-synchronizer.js

```diff
@@ -142,18 +142,17 @@ CartSynchronizer.prototype._processQueuedChanges = function () {
 };
 
 CartSynchronizer.prototype._postToServer = function ( callback ) {
-	this._wpcom.setCart(
-		this._cartKey,
-		preprocessCartForServer( this._latestValue ),
-		function ( error, newValue ) {
-			if ( error ) {
-				callback( error );
-				return;
-			}
-
-			callback( null, preprocessCartFromServer( newValue ) );
+	this._wpcom.setCart( this._cartKey, preprocessCartForServer( this._latestValue ), function (
```

Comment:
AFAICT this doesn't change anything in this file, so I assume it's just a formatting change. Was this intentional as part of this PR?